demo/src/screens/MainScreen.js

```diff
@@ -174,17 +220,80 @@ class MainScreen extends Component {
         hideUnderline
         floatingPlaceholder={false}
         text70
-        trailingAccessory={
+        leadingAccessory={
           filterText ? (
-            <Button link iconSource={Assets.icons.demo.close} $iconDefault onPress={this.clearSearch}/>
+            <Button link marginR-5 iconSource={Assets.icons.demo.close} $iconDefault onPress={this.clearSearch}/>
           ) : (
-            <Icon tintColor={Colors.$iconDefault} source={Assets.icons.demo.search}/>
+            <Icon marginR-10 tintColor={Colors.$iconDefault} source={Assets.icons.demo.search}/>
           )
         }
       />
     );
   };
 
+  onPress = ({customValue: index}) => {
+    const {chipsLabels} = this.state;
+    this.onPressFlag = true;
+    this.onUserScroll = false;
+    this.setState({
+      selectedSection: index,
+      faderStart: index != 0,
```

Comment:
always use ref comparison with double `=`, in this case use `!==` 
Btw, it should show you error lint error here, If you don't see it let's go over it and verify you environment is configured accordingly